mne/beamformer/_lcmv.py

```diff
@@ -211,44 +287,82 @@ def _apply_lcmv(data, info, tmin, forward, reg, noise_cov=None,
         W = W[2::3]
         is_free_ori = False
 
+    filters = dict(weights=W, data_cov=data_cov, noise_cov=noise_cov,
+                   whitener=whitener, weight_norm=weight_norm,
+                   pick_ori=pick_ori, ch_names=ch_names, proj=proj,
+                   is_ssp=is_ssp, vertices=vertno, is_free_ori=is_free_ori,
+                   nsource=forward['nsource'], src=deepcopy(forward['src']))
+
+    return filters
+
+
+def _subject_from_filter(filters):
+    """Get subject id from inverse operator."""
+    return filters['src'][0].get('subject_his_id', None)
+
+
+def _check_proj_match(info, filters):
+    """Check whether SSP projections in data and spatial filter match."""
+    proj_data, _, _ = make_projector(info['projs'],
+                                     filters['ch_names'])
+    if not np.array_equal(proj_data, filters['proj']):
```

Comment:
Those are numpy arrays. I think comparing the lists of info['projs'] could introduce problems if for example the data also has a set of EEG channels with a respective entry in info['projs'] but EEG channels are not picked (this is the case in the test data).